src/core/control/XournalMain.cpp

```diff
@@ -85,7 +85,7 @@ void initCAndCoutLocales() {
 void initLocalisation() {
 #ifdef ENABLE_NLS
     fs::path localeDir = Util::getGettextFilepath(Util::getLocalePath().u8string().c_str());
-    bindtextdomain(GETTEXT_PACKAGE, localeDir.u8string().c_str());
+    bindtextdomain(GETTEXT_PACKAGE, localeDir.string().c_str());
```

Comment:
How you know that `bindtextdomain` requires system narrow encoded strings instead of utf-8?

Reply:
https://github.com/microsoft/vcpkg/issues/9854

We must be careful here, vcpkg fixed this for gettext.

Reply:
> Maybe we should use `wbindtextdomain`, as suggested by the [documentation](https://www.gnu.org/software/gettext/manual/gettext.html#:~:text=On%20native%20Windows%20platforms%2C%20the%20main%20function%20may%20call%20wbindtextdomain%20instead%20of%20bindtextdomain.).

I think we are with that on the secure side. The patch does not seem to be applied upstream, only in vcpkg.